src/domains/ecommerce-app/api/ecommerce-transactions-service/v2/_base_policy.xml.tpl

```diff
@@ -0,0 +1,17 @@
+<policies>
+    <inbound>
+      <set-header name="X-Client-Id" exists-action="delete" />
+      <!--> Add here some logic for value X-Client-Id header properly such as based on api key etc <-->
+      <base />
+      <set-backend-service base-url="https://${hostname}/pagopa-ecommerce-transactions-service/v2" />
```

Comment:
is v2 required?
if we add the `v2` in the openapi file I think we can avoid it here

Reply:
Yes we can use both approach, I will change openapi descriptors :)

Reply:
I tried modifying open api path (adding preceding v2) but this way v2 prefix will be duplicated, one from api version set in api module and one from openapi path so I would leave this as is. What do you think?